test/evm/hardhat/chain-specific-spokepools/AlephZero_SpokePool.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable node/no-missing-import */
```

Comment:
I'd prefer not disabling this warning and fixing the underlying issue. Is this something we can't fix for some reason?